pkg/openstack/cinder.go

```diff
@@ -18,6 +18,7 @@ import (
 // ReconcileCinder -
 func ReconcileCinder(ctx context.Context, instance *corev1beta1.OpenStackControlPlane, helper *helper.Helper) (ctrl.Result, error) {
 	if !instance.Spec.Cinder.Enabled {
+		instance.Status.Conditions.MarkTrue(corev1beta1.OpenStackControlPlaneCinderReadyCondition, "Cinder Service is Disabled")
```

Comment:
Implementing @kajinamit suggestion here. So in cases where the service is disabled, we just set the Ready status with a message to indicate the service is disabled.